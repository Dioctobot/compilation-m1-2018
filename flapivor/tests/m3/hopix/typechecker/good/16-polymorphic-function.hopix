def id : forall a. a -> a (x) = x

val main : Int = id<Int> (37)