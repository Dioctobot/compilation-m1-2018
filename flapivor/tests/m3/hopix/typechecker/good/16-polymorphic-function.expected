id : forall a. a -> a
main : Int
